try:
    f = open('test_file.txt')
except FileNotFoundError:
    print("sorry! file not found")

except Exception:
    print("sorry! file not found")
else:
    print(f.read())
    f.close()    
finally:
    print('executing finally...')    


# or
try:
    bad_var = number

except Exception as e:
    print(e)